### ISSUE TITLE
img2sixel cannot process animated GIF created by golang's image/gif

### ISSUE THREAD

**Issue by yoichi** [saitoha/libsixel#115]
I get "corrupt GIF (reason: no clear code)" error with animated GIF created by golang's image/gif.
It can be reproduced with the last example in
http://tech.nitoyon.com/ja/blog/2016/01/07/go-animated-gif-gen/

How to reproduce:
```
$ img2sixel http://tech.nitoyon.com/ja/blog/2016/01/07/go-animated-gif-gen/rgb.gif
runtime error
corrupt GIF (reason: no clear code).
```

Expected behavior:
The animated GIF be displayed as it is displayed Chrome browser.

Environment:
```
$ img2sixel --version
img2sixel 1.8.2

configured with:
  libcurl: yes
  libpng: no
  libjpeg: yes
  gdk-pixbuf2: no
  GD: no
```

**Comment by saitoha** [comment]
f04992f in `release-1.8` branch fixes this problem.

**Comment by yoichi** [comment]
I've confirmed the fix by using 020083b from release-1.8 branch.
Thanks!

**Comment by saitoha** [comment]
Fixed on v1.8.5. Thanks!
